Features/Stories/Fieldtypes/CheckboxFieldType.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Test the validations done on fields from Editorial Interface - Checkbox fieldtype
+
+    @javascript
+    Scenario: Publish with checkbox checked
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Checkbox" in Group with identifier "Content" with fields:
```

Comment:
In BDD scenarios, 3 phases exists: pre-condition, action, validation/verification.
To be able to do the action, the pre-condition has to be filled, i.e., the content type has to exist or be created. 
To do that, 3 options might be followed: 
-- either Content Type Group is referred in the sentence so that Content Type might be created within it if it don't exist, 
-- or one can assume that content type always belong to the same content type group, i.e, "Content"
-- Or join the two previous approaches, having two different sentences: one omitting Content Type Group (assuming the "Content" is used) and other referring Content Type Group for the other situations.